pandas/tests/reshape/merge/test_merge.py

```diff
@@ -1036,11 +1037,57 @@ def test_merge_two_empty_df_no_division_error(self):
             merge(a, a, on=('a', 'b'))
 
     @pytest.mark.parametrize('how', ['right', 'outer'])
-    def test_merge_on_index_with_more_values(self, how):
+    @pytest.mark.parametrize(
+        'index,expected_index',
+        [(CategoricalIndex([1, 2, 4]),
+          CategoricalIndex([1, 2, 4, None, None, None])),
+         (DatetimeIndex(['2001-01-01',
```

Comment:
its ok to put multiple values on a line el.g. in the DTI and other construction to make this a bit shorter